gno.land/pkg/gnoland/vals.go

```diff
@@ -0,0 +1,60 @@
+package gnoland
+
+import (
+	"regexp"
+
+	gnovm "github.com/gnolang/gno/gnovm/stdlibs/std"
+	"github.com/gnolang/gno/tm2/pkg/bft/types"
+	"github.com/gnolang/gno/tm2/pkg/events"
+)
+
+const (
+	valRealm     = "gno.land/r/sys/validators"
+	valChangesFn = "GetChanges"
+
+	validatorAddedEvent   = "ValidatorAdded"
+	validatorRemovedEvent = "ValidatorRemoved"
+)
+
+var valRegexp = regexp.MustCompile(`{\("([^"]*)"\s[^)]+\),\("((?:[^"]|\\")*)"\s[^)]+\),\((\d+)\s[^)]+\)}`)
```

Comment:
@gfanton, is there a readily available better alternative?

Reply:
It's funny you should mention that, because @gfanton suggested this regex :)

We went with this approach because the JSON VM calls are still not on the horizon, and this implementation allows us to easily change it in a rolling update (since it's go)

Reply:
```suggestion
// XXX: replace with amino-based clean approach
var valRegexp = regexp.MustCompile(`{\("([^"]*)"\s[^)]+\),\("((?:[^"]|\\")*)"\s[^)]+\),\((\d+)\s[^)]+\)}`)
```